Assets/MixedRealityToolkit/_Core/Devices/Generic/XBoxController.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Internal.Definitions;
+using Microsoft.MixedReality.Toolkit.Internal.Definitions.Devices;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem;
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Devices.Generic
+{
+    // TODO - Implement
+    public struct XBoxController : IMixedRealityController
```

Comment:
Not sure about the name here.
Might just be GamepadController per windows namespace naming.

Reply:
But also, we could reuse the controller type for more than just Xbox devices

Reply:
Done